src/main/java/org/jabref/logic/integrity/CitationKeyChecker.java

```diff
@@ -11,7 +11,8 @@
 import org.jabref.model.strings.StringUtil;
 
 /**
- * Currently only checks the key if there is an author, year, and title present.
+ * In the past only checks the key if there is an author, year, and title present.
+ * Currently checks the key all the time.
```

Comment:
Past is not of interest for the current implementation. Past can be viewed in git history.